### ISSUE TITLE
Instructions are broken: unable to load

### ISSUE THREAD

**Issue by NerveCoordinator** [stisa/godot-tcp-example#1]
I tried and failed to figure out how to load this project into godot. The instructions say to `Open the project inside godot_tcp in GodotEngine,` but in `godot-tcp-example/godot_tcp`, there is no project.godot or anything else godot can detect with its 'scan' option. 

This is possibly because this project is four years old and godot changed in that time, or I might be misunderstanding something about how godot normally works.

**Comment by stisa** [comment]
> This is possibly because this project is four years old and godot changed in that time  

Yeah I think that's it, if I remember correctly this was made with godot 1.X, I guess `project.godot` is the equivalent of the old `engine.cfg`?
I might update this to work with 3.X in the future, but I'm currently too busy for that, sorry.
If you still want to open this, I would start [here](https://github.com/dploeger/godot-migrationnotes)
